minimal-notebook/notebook.conf

```diff
@@ -1,18 +1,10 @@
 [program:notebook]
 user=jovyan
```

Comment:
The minimal notebook specifies `NB_USER` and `environment` below makes use of it. I *would normally* suggest that `NB_USER` then gets propagated, but then a user only has to set `NB_USER` to `root` to elevate permissions.